tests/sentry/notifications/test_notifications.py

```diff
@@ -200,15 +201,15 @@ def test_sends_resolution_notification(self, record_analytics):
             record_analytics,
             "integrations.email.notification_sent",
             user_id=self.user.id,
-            actor_id=self.user.actor_id,
+            actor_id=Actor.objects.get(user_id=self.user.id).id,
```

Comment:
```suggestion
            actor_id=get_actor_id_for_user(self.user.id),
```

Should this be using `get_actor_id_for_user`?